catalogue/webapp/pages/item.js

```diff
@@ -293,6 +293,43 @@ const ItemPage = ({
       imageAltText={''}
       hideNewsletterPromo={true}
     >
+      <Layout12>
+        <div
+          className={classNames({
+            [spacing({ s: 4 }, { margin: ['bottom'] })]: true,
+            [spacing({ s: 6 }, { padding: ['top'] })]: true,
+          })}
+        >
+          <h1
+            className={classNames({
+              [font({ s: 'HNM3', m: 'HNM2', l: 'HNM1' })]: true,
+            })}
+            style={{
```

Comment:
A component called `SinglLineEllipsised` perhaps?
Just thinking about weening out the multiple ways of doing things inline styles being that.

Reply:
Yeah, good shout. I was thinking that this all felt like it was going to need so much design overhaul that it didn't warrant componentising, but a thing that truncates text to a single line couldn't hurt.